backend/tracim_backend/models/context_models.py

```diff
@@ -99,17 +101,17 @@ class UserCreation(object):
     def __init__(
             self,
             email: str,
-            password: str,
-            public_name: str,
-            timezone: str,
-            profile: str,
-            email_notification: str,
+            password: str = None,
+            public_name: str = None,
+            timezone: str = None,
+            profile: str = None,
+            email_notification: bool = True,
     ) -> None:
         self.email = email
-        self.password = password
-        self.public_name = public_name
-        self.timezone = timezone
-        self.profile = profile
+        self.password = password or password_generator()
```

Comment:
How is used this object ? What is set in `password` attribute ?

Reply:
this object is used in user creation endpoint, password is cleartext password. If admin set password, password is the password given by admin, else an autogenerated password is created for user (password_generator() function)

Reply:
And these data are sent as response ? My reflexion is: "Is that a good idea to give clear text password is response ?".

Reply:
**never** send passwords (clear or hashed)

Reply:
This is not response, this is user creation request.

Reply:
when user do a user creation request, this object is created with user parameters, password_generator() create a default password for user, which can be send directly by email if notification is activated according to old password notification of tracimv1 (we probably should change this mecanism but actually this is not yet done).

Reply:
Okay perfect.